lib/liquid/standardfilters.rb

```diff
@@ -270,7 +270,7 @@ def replace_first(input, string, replacement = '')
 
     # Replace the last occurrences of a string with another
     def replace_last(input, string, replacement = '')
-      input.to_s.reserve.sub(string.to_s.reserve, replacement.to_s.reserve).reserve
+      input.to_s.reverse.sub(string.to_s.reverse, replacement.to_s.reverse).reverse
```

Comment:
**Tip:** If your parent commit is local (not pushed yet), simply use `git commit --amend` to overwrite the parent commit with simple corrections like this. That way you don't create another commit just for fixing mistakes 🙂 

Even if it's pushed to the PR recently, as long as I can be reasonably sure no one is pulling the PR yet, I'd also do the same and use `git push --force` to overwrite.

And if you have multiple commits to fix, use `git rebase -i` 😄